src/bumper.js

```diff
@@ -142,6 +178,53 @@ class Bumper extends BasePlugin implements IMiddlewareProvider, IAdsControllerPr
     this._addBindings();
   }
 
+  playOnMainVideoTag(): boolean {
+    return (
+      Env.os.name === 'iOS' &&
```

Comment:
@OrenMe any suggestions on what to add here?
or maybe we should handle it by config in kaltura player?

Reply:
you mean control if use same media tag?

Reply:
yes